### DISCUSSION TITLE
pip install sqlacodegen não vai para versão atual

### DISCUSSION THREAD

**Discussion by jvras58** [agronholm/sqlacodegen#251] [General]
olá queria ajuda sobre como obter a versão atual do sqlacodegen o erro que eu to enfrentando  'ArgSpec' foi removida na versão 3.11. O módulo sqlacodegen a solução seria atualizar para essa versão porém não consigo alguem pode dar aquele help?

**Comment by agronholm** [comment]
Next time, please use English, not Portuguese or any other language. But please try installing 3.0.0rc1.
